ietf-optical-impairment-topology.yang

```diff
@@ -223,19 +255,19 @@ module ietf-optical-impairment-topology {
       config false;
       description "FEC-code-rate";
     }
-  
+
     leaf FEC-threshold {
       type decimal64 {
       fraction-digits 8;
       range "0..max";
       }
       config false;
-    description 
-      "Threshold on the BER, for which FEC is able to correct errors";
+    description
+        "Threshold on the BER, for which FEC is able to correct errors";
     }
-    
+
   }
-    
+
   grouping sliceable-transponder-attributes {

```

Comment:
The term "sliceable-transponder" is very unclear. The description does not provide more information what a sliceable transponder is. Therefore, a better term shall be used.

Reply:
Could you propose something? Suggestion feature can be used.

Reply:
I think we need more work on the transponder model (I have just created an open issue #29 )

For this version we either keep the current name or change it to `transponder-attributes`

If you change the grouping name you have also to change the code where you use this grouping